### ISSUE TITLE
how to deploy a local kernelci？ : deploynebt

### ISSUE THREAD

**Issue by guokr321** [kernelci/kernelci-project#222]
hello， I am very interested in kernelci as a kernel developer and I want to deploy a local instance of kernelCI. I have a try with https://kernelci.org/docs/instances/local/ but it does not work for some dependencies problems. Is there a detailed document for a newbie to deploy a kernelci in my own sever with the new source code?

**Comment by gctucker** [comment]
Yes, unfortunately the kernelci-backend, kernelci-frontend and kernelci-jenkins projects are now deprecated and only critical bug fixes that affect https://linux.kernelci.org are going to be merged.  They are not trivial to deploy and there is no plan to improve things there, instead they will be retired in the coming months as we're now focusing on the new API & Pipeline as a replacement for KernelCI.

Is there a particular thing you wanted to do with kernelci, or is this just out of interest?  If you want to learn and maybe contribute to KernelCI then please take a look at the new API & Pipeline, here's a quick start guide:
https://kernelci.org/docs/api/getting-started/

**Comment by nuclearcat** [comment]
Closing as answered and superseded. The legacy frontend/backend stack referenced by the original question has been retired. The thread points to the replacement API/Pipeline architecture, and the current local deployment tooling is maintained under `kernelci-deploy/localinstall`. Problems with the current tooling should be reported in the corresponding component repository.
